### DISCUSSION TITLE
Please switch to common syntax for mermaid block declaration

### DISCUSSION THREAD

**Discussion by max-mykhailenko** [outline/outline#3898] [General]
Almost all external tools understand mermaid syntax when code block starts with `mermaid` but not with `mermaidjs`. is it possible to change it?

https://mermaid-js.github.io/mermaid/#/integrations?id=editor-plugins

**Comment by naepflin** [comment]
Second this! I had to write some custom logic to overwrite my MD files to use mermaidjs instead of mermaid. I would suggest to make both mermaid and mermaidjs work for backward compatibility.

**Reply by tommoor** [reply]
Issue created: https://github.com/outline/outline/issues/11120
